src/wazuh_modules/vulnerability_detector/wm_vuln_detector.h

```diff
@@ -354,6 +356,8 @@ typedef struct scan_agent {
     vu_feed dist;
     vu_feed dist_ver;
     scan_agent_flags flags;
+    int triaged;
```

Comment:
As there are "agent related" data in this structure I think we should rename this to **os_triaged** to avoid misunderstanding the concept as the whole agent (that includes the packages) has been triaged.

Reply:
Fixed